api-reference/organization.md

```diff
@@ -8,149 +8,149 @@ An `Organization` instance represents an Aragon organization and allows to inter
 
```

Comment:
We still don't have support for several of the API methods on this document. Should we remove them from the docs now?

Reply:
Is this still necessary with our current GitBook setup?

I will go ahead and give a stab to include the few API methods we miss. I believe we already have all the logic we need.